### PR TITLE
first draft of lesson 12

### PR DESCRIPTION
**Pull request by neyoalao**
please kindly go through the draft. Add your contributions and things that needs corrections.

### CODE REVIEW

File: lesson12/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- How buttons can be styled
+- How to style borders (border radius, border style e.g dotted....)
+
+ETA: 20 mins
+
+## 2. CSS Display types
+
+students get understanding of what display types and different elements with their default display times
+
+- understand what display block, inline and inline-block
```
Comment (angelsvirkov) [inline lesson12/index.md:30]:
```suggestion
- understand what display block, inline, inline-block, and none
```

File: lesson12/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+this section will help the solidify their knowledge of the box model. they will also get to combine many things they have learnt together here.
+
+- solidify knowledge of padding and margin
+- How buttons can be styled
+- How to style borders (border radius, border style e.g dotted....)
+
+ETA: 20 mins
+
+## 2. CSS Display types
```
Comment (angelsvirkov) [inline lesson12/index.md:26]: I suggest having CSS Display types I because at later point in time we would also introduce flex and grid which are more advanced display types that we can label as `CSS Display Types II` and `CSS Display Types III`
```suggestion
## 2. CSS Display types I
```

File: lesson12/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+title: Lesson 11 - CSS Position and Display types
```
Comment (angelsvirkov) [inline lesson12/index.md:2]:
```suggestion
title: Lesson 12 - CSS Position and Display types I
```

File: lesson12/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+# CSS Icons Links and Lists
```
Comment (angelsvirkov) [inline lesson12/index.md:5]:
```suggestion
# CSS Position and Display types I
```

**Review by angelsvirkov — CHANGES_REQUESTED** [review]
Thanks for the PR @neyoalao :). Here is my feedback.

I agree that lesson 12 should include the simple css display types and the css position. I think at the end of the lecture students will be too tired and we will have too little time to cover `CSS Pseudo Elements 1`, so I suggest dropping this topic completely. Regarding the section `Styling a button`, I am not particularly sure. We actually have anchor tags that are like buttons for us and we have already included such a section in lesson 10 I think. Here is a [reference](https://github.com/ReDI-School/nrw-html-and-css-2021-spring/pull/27#issuecomment-817313873). 

So to sum up I think in this lesson we should cover
- CSS Display Types I
- CSS Position
( we have to see if we could sneak in some more things here ). Maybe a small DevTools tutorial once again to make students confident using it (could be done at the beginning of the class)

**Comment by angelsvirkov** [comment]
@neyoalao , as agreed during the task force meeting we have created the following content for lesson 13. Would you please close this PR and open a new one for lesson 13. Feel free to do this after finishing your class. Thank you. 

```
# Lesson 13 Display

1. Normal flow
2. CSS Display types
3. CSS Floats

## 1. CSS Display types

students get understanding of what display types and different elements with their default display times

Only those: `block, inline and inline-block`

- understand what display block, inline and inline-block
- how to change the default `display` of elements to align differently. E.g. `anchor` tags are by default inline but you can make them block.

- explain that height and width don't work for in-line elements. it also doesn't allow margin-top and bottom


ETA: 50min

## 3. Teach Float

Use in the context of the navigation and footer. So only these things.

ETA: 50min
```

**Comment by neyoalao** [comment]
@angelsvirkov thanks for the update. If I understand correctly, I have to elaborate on the topics and checkpoint of lesson 13 right?

**Comment by angelsvirkov** [comment]
> @angelsvirkov thanks for the update. If I understand correctly, I have to elaborate on the topics and checkpoint of lesson 13 right?

Yes exactly. The trickiest point is the checkpoint. Also we need to make sure to fill in the Glossary, Resources, and the Homework
